From: Javier Viguera <javier.viguera@digi.com>
Date: Mon, 13 Jun 2022 17:39:29 +0200
Subject: [PATCH] qcacld-3.0: disable warnings as errors

Building the kernel module under Yocto (with new version of GCC
compiler) is failing because '-Werror' is converting compilation
warnings into errors.

Signed-off-by: Javier Viguera <javier.viguera@digi.com>
---
 Kbuild | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Kbuild b/Kbuild
index ad086656cca2..50c01ef223c1 100644
--- a/Kbuild
+++ b/Kbuild
@@ -2804,7 +2804,6 @@ ccflags-y += $(INCS)
 
 cppflags-y +=	-DANI_OS_TYPE_ANDROID=6 \
 		-Wall\
-		-Werror\
 		-D__linux__
 
 cppflags-$(CONFIG_PTT_SOCK_SVC_ENABLE) += -DPTT_SOCK_SVC_ENABLE
